moveit_plugins/moveit_simple_controller_manager/CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 3.1.3)
+cmake_minimum_required(VERSION 3.10.2) # For IN_LIST policy support
```

Comment:
I increased the version number for IN_LIST policy support, I should remove the comment